App.tsx

```diff
@@ -4,17 +4,19 @@ import { Provider } from 'react-redux';
 import { config as gluestackConfig } from '@gluestack-ui/config';
 import { GluestackUIProvider } from '@gluestack-ui/themed';
 import { NavigationContainer } from '@react-navigation/native';
-
-import { AppNavigation } from '@/src/app/navigation';
-import { store } from '@/src/store';
+import { AppNavigation } from '@src/app/navigation';
+import LocalizationProvider from '@src/localization/LocalizationProvider';
```

Comment:
Also, remember that all the files must be written in `kebab-case` with lower case for all